### PR TITLE
Logging: Convert all times in log to microseconds.

### PR DESCRIPTION
**Pull request by riverszhang89**
Continue reporting LONG EVENTS, etc. as we do now, but report microseconds for event logging.

### CODE REVIEW

File: bdb/bdb_api.h [hunk 1]
```diff
@@ -135,29 +135,39 @@ struct bdb_queue_stats {
 };
 
 /* This is identical to bb_berkdb_thread_stats in db.h */
+
+/* Multiplication usually takes fewer CPU cycles than division. Therefore
+   when comparing a usec and a msec, it is preferable to use:
+   usec <comparison operator> USEC(msec)  */
+#ifndef MSEC
+#define MSEC(usec) ((usec) / 1000)
```
Comment (adizaimi) [inline bdb/bdb_api.h]: Suggestion: May it help if we rename this USEC_to_MSEC so its immediately clear what it does? (and USEC_TO_MSEC() below)

Comment (riverszhang89) [inline bdb/bdb_api.h]: Good point.

**Comment by riverszhang89** [comment]
Thank you Mike.

**Comment by riverszhang89** [comment]
Thank you Adi.

**Pull request merged**